client/components/Home.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import ProptTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import Slider from './Slider.jsx';
```

Comment:
`Nitpick`: It'll be a lot cleaner to maintain consistency with adding or not adding file extensions (`.jsx`) to import statements.